macos/Classes/FlutterLocalAuthenticationPlugin.swift

```diff
@@ -3,23 +3,23 @@ import LocalAuthentication
 import FlutterMacOS
 
 public class FlutterLocalAuthenticationPlugin: NSObject, FlutterPlugin {
-    
-    let context = LAContext()
-    
+
     public static func register(with registrar: FlutterPluginRegistrar) {
         let channel = FlutterMethodChannel(name: "flutter_local_authentication", binaryMessenger: registrar.messenger)
         let instance = FlutterLocalAuthenticationPlugin()
         registrar.addMethodCallDelegate(instance, channel: channel)
     }
 
     public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
+        let context = LAContext()
```

Comment:
Hello, thanks very much for your contribution.

Recreating an authentication context each time this function is called is not a desired behaviour. If you want to prompt authentication each time, the *touchIDAuthenticationAllowableReuseDuration* should be configurable to something different than zero.